assignments/closure.js

```diff
@@ -4,19 +4,41 @@
 // that manipulates variables defined in the outer scope.
 // The outer scope can be a parent function, or the top level of the script.
 
+var cities =['london', 'paris', 'lagos', 'rome'];
+
+const newCities = () =>{
+
+  return cities.push('Miami');
```

Comment:
You might want to return cities after you have pushed
```
cities.push('Miami')
return cities;
```
This way you return the list of cities including the newly added city
Your current logic returns the length of the cities array